app/sidekiq/hca/submission_job.rb

```diff
@@ -8,7 +8,7 @@ class SubmissionJob < BaseSubmissionJob
       health_care_application = HealthCareApplication.find(msg['args'][2])
       form = decrypt_form(msg['args'][1])
 
-      health_care_application.update!(
+      health_care_application.update(
```

Comment:
Removing the `!` causes it to not run model validations. I ran into an issue where I was manually changing a model from the rails console and it was throwing a validation error here and not call the `after_save` hooks needed to send an email. We don't currently have any validations that should be running on the update, but I don't really see a good reason to run those when we know it has already failed? Open to differing opinions.

Reply:
@dellerbie I just pushed up some changes on this. It looks like `updated_attributes` is deprecated, and `update_columns` only works for attributes that have a database column. `form` and `google_analytics_client_id` are not in the database, so that won't work here. I went ahead and set them on the object and then called `save(validate: false)` on the object which will set the values, skip validations, and trigger callbacks to continue that flow.